path_dynamic_creds.go

```diff
@@ -15,6 +16,30 @@ import (
 	"golang.org/x/text/encoding/unicode"
 )
 
+func (b *backend) pathDynamicCredsCreate() []*framework.Path {
+	return []*framework.Path{
```

Comment:
Is this function named correctly still? It seems to only have the read operation defined.

Reply:
I _think_ it makes sense. The read operation in this case creates an LDAP user and secret in Vault ([path_dynamic_creds.go#L18-L92](https://github.com/hashicorp/vault-plugin-secrets-openldap/blob/main/path_dynamic_creds.go#L18-L92)). One of those Vault-y things where a read creates something 🙂